### PR TITLE
updated build:production to use build script directly

### PR DESCRIPTION
**Pull request by johanna-skylight**
# DEVOPS PULL REQUEST

## Related Issue

- Resolves

## Changes Proposed

- Sets the build:production script that creates the prod bundle to run the build directly, skipping the `func extensions install ` that should only be used for local setup.
- Updated failing frontend unit test due to trying to mock a readOnly variable by accessing the fields directly.

## Additional Information

None

## Testing

- Tested in dev6 and dev2. Functionality to report to the covid pipeline, universal pipeline and processing the publishing errors worked as expected.

### CODE REVIEW

**Review by BobanL — APPROVED** [review]
Tested in dev6 and the function app works as intended!

**Review by emyl3 — APPROVED** [review]
Tested covid and flu tests and saw them being added to the queue! Was able to run `yarn build:localSetup` successfully.

File: frontend/src/app/VersionService.test.tsx
Comment (johanna-skylight) [reply frontend/src/app/VersionService.test.tsx:93]: This unit testing also started to failed and is blocking the PRs as well. Changing the global element directly is unpredictable as it is a read only element, so the Object.defineProperty should be used instead.

Comment (zdeveloper) [reply frontend/src/app/VersionService.test.tsx:93]: in that case, the test fix doesnt need to wait on this PR, could be merged separately

Comment (johanna-skylight) [reply frontend/src/app/VersionService.test.tsx:93]: But this PR is already to be merged unless you have any concerns to delay it 😅

**Review by zdeveloper — APPROVED** [review]
LGTM! made implementation changes and was able to verify that it was deployed on dev6. Please update the PR description before merging to document the VersionService test changes

File: frontend/src/app/VersionService.test.tsx
Comment (zdeveloper) [reply frontend/src/app/VersionService.test.tsx:93]: so I added an implementation change on top of this branch, and tested that I was able to see the new implementation in dev6. The idea is to verify that the newer build is actually updating the implementation and not using a cached version

**Pull request merged**
